packages-exp/app-compat/package.json

```diff
@@ -24,7 +24,8 @@
     "test:browser": "karma start --single-run",
     "test:browser:debug": "karma start --browsers Chrome --auto-watch",
     "test:node": "TS_NODE_FILES=true TS_NODE_CACHE=NO TS_NODE_COMPILER_OPTIONS='{\"module\":\"commonjs\"}' nyc --reporter lcovonly -- mocha test/**/*.test.* src/**/*.test.ts --config ../../config/mocharc.node.js",
-    "prepare": "yarn build:release"
+    "prepare": "yarn build:release",
+    "api-report": "api-extractor run --local --verbose && ts-node-script ../../repo-scripts/prune-dts/prune-dts.ts --input exp/dist/storage-public.d.ts --output exp/dist/storage-public.d.ts"
```

Comment:
Is it the wrong package? it's app-compat's package.json.

Reply:
This was probably a dead-end from trying to debug some api-extractor warnings about imports from `app`. Doesn't seem useful, removed.